spec/models/income_spec.rb

```diff
@@ -1,5 +1,55 @@
 require 'rails_helper'
 
+def income_for_team(team_name)
+  team = Team.find_by_team_name(team_name)
+  Income.find_by(team: team)
+end
+
 RSpec.describe Income, :type => :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  Game::COUNTRIES.each { |country| Team.find_or_create_by(team_name: country) }
```

Comment:
You should just make the Team call in `before(:each)` be a `find_or_create_by` and delete this line.  Otherwise, you may cause some unintended side effect since rails does not know how to clean up the created Teams after the rspec is run.  In this case, you lucked out because we are doing a `find_or_create` but if another test eventually attempts to mess with Teams/Team creation we might have unexpected errors.

You could also do this in a `before(:all)` and then make an `after(:all)` hook that destroys all Teams.  But really it's not worth it because we don't have that many examples that its slowing down the tests.